addons/html_editor/static/src/main/media/image_description.xml

```diff
@@ -6,18 +6,20 @@
     </t>
 
     <t t-name="html_editor.ImageDescriptionDialog">
-        <Dialog size="'lg'" title.translate="Change media description and tooltip">
+        <Dialog size="'md'" title.translate="Image description">
             <div class="mb-3 row">
-                <label class="col-md-3 col-form-label" for="alt" title="'Alt tag' specifies an alternate text for an image, if the image cannot be displayed (slow connection, missing image, screen reader ...).">
```

Comment:
@lebl-odoo Was it intended that the` title` attribute for "Alt" was changed? Same for "Title" below. I think the new ones are less helpful than the previous ones.

Reply:
@dmo-odoo The main idea is to be coherent between the editor and the website builder. Here a screenshot of the option in the builder:
![image](https://github.com/user-attachments/assets/714b150c-1f22-4173-b225-066b915a9b6b)

I must have been drunk when I wrote those specs :roll_eyes: 
* We should keep "Tooltip" instead of "Title"
* Tooltip: "'Title tag' is shown as a tooltip when you hover the picture."
* Description: "Alt tag' specifies an alternate text for an image, if the image cannot be displayed (slow connection, missing image, screen reader ...)."

My bad @adch-odoo, thx for noticing it @dmo-odoo

Reply:
Done with changes.